frontend/fingertips-frontend/components/organisms/ThematicMap/index.tsx

```diff
@@ -97,18 +32,19 @@ export function ThematicMap({
   // causes it to loop infinitely. (https://react.dev/reference/react/useEffect#examples-dependencies)
   useEffect(() => {
     loadHighchartsModules(async () => {
-      setOptions(mapOptions);
+      setOptions(createThematicMapChartOptions(mapData, healthIndicatorData));
     });
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  // This prevents errors from trying to render before the module is loaded in the useEffect callback
   if (!options) {
     return null;
   }
 
   return (
     <div data-testid="thematicMap-component">
-      <H3>{mapTitle}</H3>
+      <BenchmarkLegend rag={true} />
```

Comment:
```suggestion
      <BenchmarkLegend rag />
```
Sorry, a little pedantic, although I notice the screenshots show the quintile colours and not just the rag ones?

Reply:
Good spot - I'm updating the screenshots as well.